### PR TITLE
Removes juggling

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

- Removes juggling by using a var on the mob who fired which tracks when the next shot of the gun they fired should be possible (current world time + firing delay of the weapon). If the time isn't up we arent allowed to fire any new guns. 

## Why It's Good For The Game

I should probably write a tirade about how terrible juggling is for the game, but I'll spare you. 

Basically being able to bypass the balancing of slugs (and shotguns in general) of a slow fire rate is not good.

https://user-images.githubusercontent.com/22431091/187104113-70a4b6f0-e539-4706-a436-3a79583272f9.mp4

## Changelog
:cl:
balance: Juggling weapons is no longer possible, see #10848 for details
/:cl:

### CODE REVIEW

**Comment by Lumipharon** [comment]
Does this fuck over attachment weapons?

**Comment by ivanmixo** [comment]
Good question

**Comment by Lumipharon** [comment]
Looks good with the changes.

**Comment by TiviPlus** [comment]
https://github.com/tgstation/TerraGov-Marine-Corps/pull/10852#issuecomment-1231960123
